lib/Util/PTAStat.cpp

```diff
@@ -107,12 +107,27 @@ const char* PTAStat:: NumOfNullPointer = "NullPointer";	///< Number of pointers
 
 PTAStat::PTAStat(PointerAnalysis* p) : startTime(0), endTime(0), pta(p)
 {
-
+    assert((Options::ClockType == ClockType::Wall || Options::ClockType == ClockType::CPU)
+           && "PTAStat: unknown clock type!");
 }
 
 double PTAStat::getClk(bool mark) {
     if (Options::MarkedClocksOnly && !mark) return 0.0;
```

Comment:
Why do we need both `MarkedClocksOnly ` and  `mark` here? Can we make `mark` as the default parameter `getClk(bool mark=true)`?